recipes/sphinx-basic-ng/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sphinx-basic-ng" %}
+{% set version = "0.0.1a11" %}
```

Comment:
This seems like an alpha build. In that case you need to target the pre release channel: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
thanks @BastianZim -- updated

Reply:
Great! Please ping me once the CI passes and I'll merge it.